endpoint/endpointComp/Component.cdef

```diff
@@ -10,60 +10,6 @@ sources:
 
     ${CURDIR}/../../output_base/external/http_parser/http_parser.c
 
-    ${CURDIR}/../../output_base/external/mbedtls_2_16_6/library/aes.c
```

Comment:
Since you removed the build rules about Mbed TLS, you shall explain within commit message(s).